src/components/Topmenu.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {NavLink} from 'react-router-dom';
+
+function Topmenu({isAuth, toggleAuth}) {
+    return (
+        <nav>
+            <ul className="navbar">
+                <li>
```

Comment:
Ik zie in de antwoorden dat alles in li elementen omwikkeld hoort. Netjes gedaan dus! Dat zal de norm zijn bij navigatie balken. Het is al een tijdje geleden dat ik met CSS bezig ben geweest het zal daarvoor nodig zijn.